SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -310,7 +311,31 @@ void USpatialNetDriver::CreateServerWorkerEntity()
 	Components.Add(EntityAcl(WorkerIdPermission, ComponentWriteAcl).CreateEntityAclData());
 	Components.Add(interest.CreateInterestData());
 
-	Connection->SendCreateEntityRequest(MoveTemp(Components), nullptr);
+	Worker_RequestId RequestId = Connection->SendCreateEntityRequest(MoveTemp(Components), nullptr);
+	CreateEntityDelegate OnCreateWorkerEntityResponse;
+	OnCreateWorkerEntityResponse.BindLambda([this](const Worker_CreateEntityResponseOp& Op)
+	{
+		if (Op.status_code != WORKER_STATUS_CODE_SUCCESS)
+		{
+			if (++CreateServerWorkerEntityNumberOfAttempts <= SpatialConstants::MAX_NUMBER_COMMAND_ATTEMPTS)
```

Comment:
This logic gives MAX_NUMBER_COMMAND_ATTEMPTS + 1, should be < instead?

Reply:
Also I generally don't like things like postfix increments in conditionals because I find them easy to mess up or just harder to parse that just checking then incrementing, but that's just a personal pref, no requirement to change.